internal/resourcelist/resourcelist_test.go

```diff
@@ -168,6 +168,7 @@ func TestSubCoreResources(t *testing.T) {
 	for _, tc := range testCases {
 		t.Run(ToString(tc.expected), func(t *testing.T) {
 			res := tc.res.DeepCopy()
+			//nolint:errcheck
```

Comment:
We are we choosing to make the linter happy by doing this rather than returning error and ensuring that the expected error is nil?

Reply:
you're right 
thanks for the comment